rust/actyx/ax-core/src/runtime/operation/validation.rs

```diff
@@ -321,16 +351,26 @@ mod test {
             builder.encode_u64(10).encode_u64(100);
         });
         let cbor = cbor.decode();
-        assert!(validate_array(&cbor).is_ok());
+        validate_array(&cbor, &Type::Atom(TypeAtom::Number(None))).unwrap();
+        assert!(validate_array(&cbor, &Type::Atom(TypeAtom::Number(None))).is_ok());
```

Comment:
these two do the same; I prefer the first version because upon failure I can immediately see the cause — I very rarely use `assert!()`, and if I do, I never do it with just one argument (I also never trust a test without having made it fail, which also shows me the quality of the failure output)

Reply:
This was a leftover of a debug session. Ironically, for the reason you meant. I guess it's time I rewrite them 😅

Reply:
I tried both approaches and I find the `assert!` with a second parameter better than the first. 
The first does provide much more output but IMO it doesn't help, having a short message explaining the assumption that was broken and the respective line works out better (in this case).